services/web-app/package.json

```diff
@@ -51,7 +52,7 @@
     "http-proxy-middleware": "^2.0.6",
     "js-yaml": "^4.1.0",
     "lodash": "^4.17.21",
-    "lodash-es": "^4.17.21",
+    "markdown-it": "^13.0.1",
```

Comment:
Is this package being used? I did a quick search in the code and didn't see any hits.

Reply:
I'm confused, I don't see this in my PR changes?  We're using this package in glossary-list.js which is already on main
<img width="1469" alt="image" src="https://user-images.githubusercontent.com/40550942/167451659-975a8cdf-035e-4949-b6f8-baabcee07ab2.png">

Reply:
oh it must be merge changes from another PR. Sorry!